hbase-resource-bundle/src/main/resources/META-INF/LICENSE.vm

```diff
@@ -1381,7 +1381,7 @@ You can redistribute it and/or modify it under either the terms of the
 ## See this FAQ link for justifications: https://www.apache.org/legal/resolved.html
 ##
 ## NB: This list is later compared as lower-case. New entries must also be all lower-case
-#set($non_aggregate_fine = [ 'public domain', 'new bsd license', 'bsd license', 'bsd', 'bsd 2-clause license', 'mozilla public license version 1.1', 'mozilla public license version 2.0', 'creative commons attribution license, version 2.5' ])
+#set($non_aggregate_fine = [ 'public domain', 'new bsd license', 'bsd license', 'bsd', 'bsd 2-clause license', 'mozilla public license version 1.1', 'mozilla public license version 2.0', 'creative commons attribution license, version 2.5', 'apache license 2.0' ])
```

Comment:
I'm surprised we need this change.

@busbey ?

Reply:
We definitely need it or the build fails with an error generating the LICENSE file. I also tried a supplemental model approach but that did not help.

Reply:
Mind posting the errors? In general I do not think we should add apache license 2.0 here...

Reply:
this should not change. please correct the name of the license via supplemental models.

Reply:
@busbey I tried the supplimental models approach and it did not work. I will put it back so you can observe the failure. Or perhaps by doing it again I will find some detail missed last time. I would hate to abandon this work because of this nit.

Reply:
I'm glad you had me do it again because this time I was successful.